### DISCUSSION TITLE
Ability to default envs

### DISCUSSION THREAD

**Discussion by syahnur197** [usebruno/bruno#2394] [Ideas]
Would be nice if we have the ability to specify the default the env

for example

in our .bru file

```
vars {
    url {{process.env.LOCAL_URL ?? 'https://locahost'}}
}
```

This will ensure the request still work even though if the variable is missing in .env file.

**Comment by peteruithoven** [comment]
Related: 
- https://github.com/usebruno/bruno/issues/1598
- https://github.com/usebruno/bruno/issues/2170
